build.sh

```diff
@@ -18,10 +18,15 @@
 
 SCRIPT_PATH="$(cd "$(dirname "$0")" && pwd -P)"
 
+# Build assets
+${SCRIPT_PATH}/asset.sh
```

Comment:
Yeah, I did it, and it seems to be fine! Tested with some forked+customized repo :)